llama_index/node_parser/extractors/metadata_extractors.py

```diff
@@ -407,7 +407,7 @@ class SummaryExtractor(MetadataFeatureExtractor):
     Args:
         llm_predictor (Optional[BaseLLMPredictor]): LLM predictor
         summaries (List[str]): list of summaries to extract: 'self', 'prev', 'next'
-        prompt_template (str): template for summary extraction
+        prompt_template (str): template for summary extraction.
```

Comment:
Hmm. Should this actually be a period here?

Reply:
Nice catch, yeah seems to be an edge case `ruff` got confused by.  I manually fixed this one by adding a new line above "Args", and now `ruff` works as expected